sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -109,7 +109,7 @@ export const ConnectionContextBase = {
    */
   create(parameters: CreateConnectionContextBaseParameters): ConnectionContextBase {
     ConnectionConfig.validate(parameters.config, {
-      isEntityPathRequired: parameters.isEntityPathRequired || false
+      isEntityPathRequired: parameters.isEntityPathRequired || false,
```

Comment:
Didn't realize that it does this for single entry but as long as `npm run format` fixes things for me I am fine with it.

Reply:
`npm run format` will do it, there were no manual changes on this PR except for the versions in the package.json 🙂